instaseis/scripts/repack_db.py

```diff
@@ -490,7 +490,11 @@ def repack_database(input_folder, output_folder, contiguous,
 
     assert found_filenames, "No files named `ordered_output.nc4` found."
 
-    os.makedirs(output_folder)
+    if input_folder == output_folder:
+        if "ordered_output.nc4" in found_filenames:
```

Comment:
`found_filenames` contains the full paths so this conditional will likely never be true.

This should work: 

```python
if "ordered_output.nc4" in [os.path.basename(_i) for _i in found_filenames]:
   ...
```